wandb/sdk/launch/agent/agent.py

```diff
@@ -136,6 +136,12 @@ def __init__(self, api: Api, config: Dict[str, Any]):
         self._secure_mode = config.get("secure_mode", False)
         self.default_config: Dict[str, Any] = config
 
+        # Get agent version from env var if present, otherwise wandb version
+        self.version: str = wandb.__version__
+        env_agent_version = os.environ.get("WANDB_AGENT_VERSION")
+        if env_agent_version and env_agent_version != "wandb-launch-agent":
+            self.version = env_agent_version
+
```

Comment:
This is the important part of the PR, want to make sure the logic here is what we want to go with: use the `WANDB_AGENT_VERSION` env var if present, `wandb.__version__` if not

Reply:
`wandb` version doesn't really have anything to do with `WANDB_AGENT_VERSION`, does it?

Reply:
My thinking was that if someone isn't using an agent to run an image, i.e. runs `wandb launch-agent` from their local machine, they should have some indicator of what version of the agent code they're working with. And that would be determined by wandb's version.

Reply:
I see.  maybe we should prefix the version string with a specifier e.g. `wandb@0.15.7` in the case where we fall back to the `wandb` version?  I want to avoid the situation where we persist this value and it becomes difficult to ascertain if it's a `wandb` or actual agent version without maintaining a LUT/heuristics

Reply:
I like that much better! Just pushed.